city_scrapers/spiders/chi_ssa_69.py

```diff
@@ -0,0 +1,130 @@
+            "key=AIzaSyC-KzxSLmmZitsCVv2DeueeUxoVwP0raVk"
+        ).format(
+            current_chicago_time_gmt_offset,
+            last_week.strftime("%Y-%m-%d"), current_chicago_time_gmt_offset,
+            in_two_months.strftime("%Y-%m-%d"), current_chicago_time_gmt_offset
+        )]
+
+    def parse(self, response):
+        data = json.loads(response.text)
+
+        for item in data["items"]:
+            title = self._parse_title(item)
+            location = self._parse_location(item)
+            if not location:
+                continue
+            meeting = Meeting(
+                title=title,
+                description=self._parse_description(item),
+                classification=self._parse_classification(title),
+                start=self._parse_dt(item["start"]),
+                end=self._parse_dt(item["end"]),
+                time_notes="",
+                all_day=self._parse_all_day(item),
+                location=location,
+                links=[],
+                source=item["htmlLink"],
+            )
+            meeting['status'] = self._get_status(meeting, text=item["status"])
+            meeting['id'] = self._get_id(meeting)
+            yield meeting
+
+    def _parse_title(self, item):
+        return re.sub(r" Meeting$", "", item["summary"].strip())
+
+    def _parse_classification(self, title):
+        if "committee" in title.lower():
+            return COMMITTEE
+        elif "focus group" in title.lower():
+            return FORUM
+        elif "commission" in title.lower():
+            return COMMISSION
+        return NOT_CLASSIFIED
+
+    def _parse_dt(self, dt_obj):
+        if "dateTime" in dt_obj:
+            return datetime.strptime(dt_obj["dateTime"][:19], "%Y-%m-%dT%H:%M:%S")
+        elif "date" in dt_obj:
+            return datetime.strptime(dt_obj["date"], "%Y-%m-%d")
+
+    def _parse_location(self, item):
+        if "location" not in item:
+            return
+        split_loc = re.split(r"(?<=[a-z]), (?=\d)", item["location"])
+        name = ""
+        if len(split_loc) == 1:
+            address = split_loc[0]
+
+            # Sometimes they put a room name in the location field and
+            # and address in the description, so fix that
+            if (
+                "Chicago" not in address and "Il" not in address
+                and "006th district police station" in address
+            ):
+                name = address
+                lines_in_desc = re.split(r"\n", item["description"])
+                for line in lines_in_desc:
+                    if "Chicago" in line:
+                        address = line
+
+        else:
+            name = split_loc[0]
+            address = ", ".join(split_loc[1:])
+        return {
+            "name": name,
+            "address": address,
+        }
+
+    def _parse_description(self, item):
+        """Parse or generate meeting description."""
```

Comment:
It should be fine to leave the description as is, but a simpler way of doing a null check is just to do `item.get("description", "")` which will return an empty string if the key isn't found

Reply:
Thanks for the tip. I incorporated that.